tasks.py

```diff
@@ -1,17 +1,79 @@
-import os
+import sys
+from pathlib import Path
 
 from invoke import task
+from rellu import initialize_labels, ReleaseNotesGenerator, Version
+from rellu.tasks import clean
 from robot.libdoc import libdoc
 
-assert os.getcwd() == os.path.dirname(os.path.abspath(__file__))
+
+assert Path.cwd() == Path(__file__).parent
+
+
+REPOSITORY = 'robotframework/SeleniumLibrary'
+VERSION_PATH = Path('src/SeleniumLibrary/__init__.py')
+RELEASE_NOTES_PATH = Path('docs/SeleniumLibrary-{version}.rst')
+RELEASE_NOTES_TITLE = 'SeleniumLibrary {version}'
+RELEASE_NOTES_INTRO = '''
+SeleniumLibrary_ is a web testing library for `Robot Framework`_ that utilizes
+the Selenium_ tool internally. SeleniumLibrary {version} is a new release with
+**UPDATE** enhancements and bug fixes. **ADD more intro stuff...**
+
+**REMOVE this section with final releases or otherwise if release notes contain
+all issues.**
+All issues targeted for SeleniumLibrary {version.milestone} can be found
+from the `issue tracker`_.
+
+**REMOVE ``--pre`` from the next command with final releases.**
+If you have pip_ installed, just run
+``pip install --pre --upgrade robotframework-seleniumlibrary``
+to install the latest release or use
+``pip install robotframework-seleniumlibrary=={version}``
+to install exactly this version. Alternatively you can download the source
+distribution from PyPI_ and install it manually.
+
+Rellu {version} was released on {date}.
```

Comment:
Rellu -> SeleniumLibrary

Reply:
Guess did I copy this from Rellu's tasks.py? =)

I can fix this later today or you can merge and fix yourself.